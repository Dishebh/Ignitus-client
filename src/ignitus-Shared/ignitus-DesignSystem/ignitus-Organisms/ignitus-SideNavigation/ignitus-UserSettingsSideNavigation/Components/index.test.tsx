import React from 'react';
import { mount, render } from 'enzyme';

import UserSettingsSideNavigation, { Layer } from './index';

let wrapper;
beforeEach(() => {
  wrapper = mount(<UserSettingsSideNavigation />);
});

describe('<DefaultButtonWithIcon />', () => {
  it('should rendered', () => {
    const tree = render(<UserSettingsSideNavigation />);
    expect(tree).toMatchSnapshot();
  });

  it('should rendered all layers', () => {
    expect(wrapper.find(Layer).length).toEqual(6);
  });
});
