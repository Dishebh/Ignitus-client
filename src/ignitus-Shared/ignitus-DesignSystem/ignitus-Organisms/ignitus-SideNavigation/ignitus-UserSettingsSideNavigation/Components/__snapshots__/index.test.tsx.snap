// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefaultButtonWithIcon /> should rendered 1`] = `
<div
  class="css-1xk2vj9"
>
  <div
    class="css-4l4868"
  >
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      General
    </p>
  </div>
  <div
    class="css-14ts9o7"
  >
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Password
    </p>
  </div>
  <div
    class="css-14ts9o7"
  >
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Messaging
    </p>
  </div>
  <div
    class="css-14ts9o7"
  >
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Notification
    </p>
  </div>
  <div
    class="css-14ts9o7"
  >
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Blocked Users
    </p>
  </div>
  <div
    class="css-naeija"
  >
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Privacy
    </p>
  </div>
</div>
`;
